mmv1/products/datastream/ConnectionProfile.yaml

```diff
@@ -36,12 +36,16 @@ examples:
     vars:
       connection_profile_id: 'my-profile'
   - !ruby/object:Provider::Terraform::Examples
-    name: 'datastream_connection_profile_bigquery_private_connection'
+    name: 'datastream_connection_profile_postgresql_private_connection'
     primary_resource_id: 'default'
     vars:
       private_connection_id: 'my-connection'
       connection_profile_id: 'my-profile'
       network_name: 'my-network'
+      database_instance_name: 'my-instance'
+      deletion_protection:
+        'true'
```

Comment:
We'll need to set var overrides to `false` so the test / oics can succeed in deletion: https://github.com/GoogleCloudPlatform/magic-modules/blob/905133647a32023e4aab64b0336b3f265f563295/mmv1/products/sql/Database.yaml#L42C1-L45

That will put `true` in the docs, and `false` in other spots.

Reply:
done